README.md

```diff
@@ -74,6 +74,65 @@ manager = TransferManager(client, config)
 manager.download_read_set(SEQUENCE_STORE_ID, "<my-read-set-id>")
 ```
 
+## Using the Omics URI Parser
+### Basic Usage
+The `uri_parser` class makes it easy to parse omics readset and reference URIs to extract fields relevant for calling 
```

Comment:
Looks like the name of the class is `OmicsUriParser`, and `uri_parser` is the name of the package.